Source/WebCore/Modules/websockets/ThreadableWebSocketChannelClientWrapper.cpp

```diff
@@ -257,7 +257,7 @@ void ThreadableWebSocketChannelClientWrapper::processPendingTasks()
     if (!m_syncMethodDone) {
         // When a synchronous operation is in progress (i.e. the execution stack contains
         // WorkerThreadableWebSocketChannel::waitForMethodCompletion()), we cannot invoke callbacks in this run loop.
-        m_context.postTask([this, protectedThis = Ref { *this }] (ScriptExecutionContext& context) {
+        m_context->postTask([this, protectedThis = Ref { *this }] (ScriptExecutionContext& context) {
```

Comment:
We should store m_context in RefPtr before calling this function. And ditto for other code in this patch.